.buildkite/pipeline.yml

```diff
@@ -109,6 +100,25 @@ steps:
           slurm_gres: "gpu:1"
           slurm_mem: 20GB
 
+  - group: "ClimaEarth test"
+    steps:
+      - label: "ClimaEarth test"
+        key: "restarts"
+        command: "julia --color=yes --project=experiments/ClimaEarth/ experiments/ClimaEarth/test/runtests.jl"
+        agents:
+          slurm_mem: 16GB
+
+      - label: "MPI restarts"
+        key: "mpi_restarts"
+        command: "srun julia --color=yes --project=experiments/ClimaEarth/ experiments/ClimaEarth/test/restart.jl"
+        env:
+          CLIMACOMMS_CONTEXT: "MPI"
+        timeout_in_minutes: 40
+        soft_fail: true         # The MPI test sometimes hangs
```

Comment:
Is this hanging because of code introduced here? Is this something that should be fixed?

Reply:
This is hanging for the same reason the equialent test in ClimaAtmos hangs: sometimes, things hang when multiple Atmos simulations are started in the same REPL (with MPI).

I tried debugging/working around this in the past (e.g., creating one MPI communicator per simulation), and I tried again, but I could not figure out what's wrong. 

I would like to see this fixed.